docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,78 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+The Debugging Interface will help users to debug Bear code and step through
+it using the pdb interface.
+
+Details
+-------
```

Comment:
hm earlier that looked like that there would be more information here :sweat_smile: So the text isn't really worth the "Details" paragraph :P Just remove this paragraph header completely, so the overview text above flows directly into your "details" here :P